examples/musig.c

```diff
@@ -156,7 +170,25 @@ int sign(const secp256k1_context* ctx, struct signer_secrets *signer_secrets, st
             return 0;
         }
     }
-    return secp256k1_musig_partial_sig_agg(ctx, sig64, &session, partial_sigs, N_SIGNERS);
+
+    /* Since we are doing adaptor sig, complete pre-signature */
+    if (!secp256k1_musig_nonce_parity(ctx, &nonce_parity, &session)) {
+        return 0;
+    }
+    if (!secp256k1_musig_partial_sig_agg(ctx, presig, &session, partial_sigs, N_SIGNERS)){
+        return 0;
+    }
+    if (!secp256k1_musig_adapt(ctx, sig64, presig, adaptor_key, nonce_parity)) {
+        return 0;
+    }
+	/* With sig64 "on-chain" now, other party can grab the revealed adaptor secret */
+    if (!secp256k1_musig_extract_adaptor(ctx, extracted_adaptor, sig64, presig, nonce_parity)) {
+        return 0;
+    }
+    if (memcmp(extracted_adaptor, adaptor_key, sizeof(adaptor_key)) != 0) {
```

Comment:
Can you `assert` this? We normally use `assert` for these comparisons in the other examples.